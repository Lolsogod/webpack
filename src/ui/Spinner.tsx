import styles from "@/styles/spinner.module.scss";

const Spinner = () => {
    return (
        <div className={styles.spinnerContainer} data-testid="spinner">
            <div className={styles.spinner}></div>
        </div>
    );
};

export default Spinner;

