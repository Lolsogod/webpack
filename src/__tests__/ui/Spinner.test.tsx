import React from "react";
import Spinner from "@/ui/Spinner";
import { render } from "@testing-library/react";

describe('Spinner', () => {
  it('renders correctly', () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});