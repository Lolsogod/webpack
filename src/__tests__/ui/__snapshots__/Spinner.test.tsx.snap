// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<div>
  <div
    class="spinnerContainer"
    data-testid="spinner"
  >
    <div
      class="spinner"
    />
  </div>
</div>
`;
